Source/Events.Store.Services.Grpc/EventStoreGrpcService.cs

```diff
@@ -17,14 +19,14 @@ namespace Dolittle.Runtime.Events.Store.Services.Grpc;
 [PrivateService]
 public class EventStoreGrpcService : EventStoreBase
 {
+    const uint MaxBatchMessageSize = 2097152; // 2 MB
     readonly IEventStore _eventStore;
-    readonly IFetchCommittedEvents _committedEventsFetcher;
+    readonly StreamOfBatchedMessagesSender<FetchForAggregateResponse, Contracts.CommittedAggregateEvents.Types.CommittedAggregateEvent> _aggregateEventsBatchSender = new();
```

Comment:
In the Projection service - this is a dependency on the interface. I think this could be a static class instead and then we don't need to do this differently on the two services.